src/backend/src/utils/reimbursement-requests.utils.ts

```diff
@@ -203,16 +203,58 @@ const createNewProducts = async (products: ReimbursementProductCreateArgs[], rei
 };
 
 export const validateUserIsPartOfFinanceTeam = async (user: User) => {
+  const userAuthorized = await isUserOnFinanceTeam(user);
```

Comment:
Similar to the comment in the service file, can we rename this to `isUserAuthorized`?